alibi/explainers/anchor_image.py

```diff
@@ -221,13 +221,12 @@ def perturbation(
         segments_mask[:, anchor] = 1
 
         # for each sample, need to sample one of the background images if provided
-        if self.images_background:
+        if self.images_background is not None:
```

Comment:
`self.images_background` is either `None` or `np.ndarray` so we must check explicitly for `None` as `np.ndarray` doesn't have a working `__bool__` method.